velox/type/Timestamp.cpp

```diff
@@ -166,9 +165,24 @@ void appendSmallInt(int n, std::string& out) {
   out.append(intToStr[n], 2);
 }
 
+std::string::size_type getCapacity(const TimestampToStringOptions& options) {
+  int width = static_cast<int>(options.precision);
+  switch (options.mode) {
+    case TimestampToStringOptions::Mode::kDateOnly:
+      return 10;
+    case TimestampToStringOptions::Mode::kTimeOnly:
+      // hh:mm:ss.precision
+      return 9 + width;
+    case TimestampToStringOptions::Mode::kFull:
+      return 26 + width;
```

Comment:
Please can you explain the formatting behind 26

Reply:
I don't know how to calculate this 26. Before the modification, this block was written as 26. In order not to affect the existing running queries, I did not modify it here. Maybe @Yuhta knows how this is calculated.